core-bundle/tests/Cache/ContaoCacheWarmerTest.php

```diff
@@ -46,8 +47,16 @@ protected function tearDown(): void
 
     public function testCreatesTheCacheFolder(): void
     {
+        $resourceLocator = $this->createMock(FileLocator::class);
+        $resourceLocator
+            ->method('locate')
+            ->with('config/database.sql', null, false)
+            ->willThrowException(new FileLocatorFileNotFoundException())
```

Comment:
Why not `->willReturn([])`?

Reply:
Because that's also what Symfony's file locator does if the file is missing. See https://github.com/symfony/symfony/blob/950e1444cb53780543beb4968e83f13679bd5983/src/Symfony/Component/Config/FileLocator.php#L43-L45